src/decorateUIRouterViewConfigs.ts

```diff
@@ -29,9 +29,10 @@ hybridModule.config(['$uiRouterProvider', (router: UIRouter) => {
 
     Object.keys(views).forEach(key => {
       const view = views[key];
-      const selfView = selfViews[key];
+      // Account for views: { "": { template... } }
+      const selfView = selfViews[key] || selfViews['$default'];
```

Comment:
@christopherthielen i'm wondering if i should go back to my angular routes and look for definitions that have views: "" and update it to "$default", or is this still a supported feature?

Reply:
This is still supported.  Thanks for the PR!  I think we'll change it to 
```
 const selfView = selfViews[key || '$default'];
```